packages/contracts-bedrock/src/interop/CrossL2Inbox.sol

```diff
@@ -47,4 +66,88 @@ contract CrossL2Inbox is ISemver {
             roots[mail[i].chain][mail[i].output] = true;
         }
     }
+
+    /// @notice Verifies and executes a cross-L2 message.
+    /// @param _msg            Cross L2 message to finalize.
+    /// @param _l2OutputRoot   Cross L2 outbox root to prove against. Only previously posted output roots are accepted.
+    /// @param _inclusionProof Inclusion proof of the CrossL2Outbox contract's storage root.
+    function runCrossL2Message(
+        Types.SuperchainMessage memory _msg,
+        bytes32 _l2OutputRoot,
+        bytes calldata _inclusionProof
+    )
+        external
+    {
+        // TODO: should check _msg.to to not get round-trip inbox/outbox interactions
+        // that have the system contract address as _msg.from.
+
+        require(
+            roots[_msg.sourceChain][_l2OutputRoot] != bytes32(),
```

Comment:
til: you don't need to do `bytes32(0)`

Reply:
Compiler seemed happy, but seeing other errors in CI elsewhere, so it may have been suppressed because of that. Based on `chisel` it actually seems to need the `0`.